Makefile

```diff
@@ -36,6 +36,15 @@ docs-lint: ## Check documentation for common syntax errors.
 # The `-n` option enables "nit-picky" mode.
 	make -C docs/ clean && sphinx-build -Wn docs/ docs/_build/html
 
+.PHONY: update-user-guides
+update-user-guides: ## Update screenshots for the user guides.
+	if [ -d "/vagrant" ]; then \
+			bash -c "pushd /vagrant/securedrop; pytest -v tests/pages-layout --page-layout; popd"; \
+			cp /vagrant/securedrop/tests/pages-layout/screenshots/en_US/*.png /vagrant/docs/images/manual/; \
+	else \
+      printf "You must run this from the development VM!\n"; \
+	fi
+
```

Comment:
Nit: the `printf` line is indented with space, when it should be indented with two tabs.

Reply:
Woops - nits fixed. Sorry about that 😞